www/src/components/banner.js

```diff
@@ -45,9 +45,18 @@ const Banner = () => (
     <InnerContainer>
       <Content>
         {`New! Try Incremental Builds with `}
-        <OutboundLink href="https://www.gatsbyjs.com">
+        <a
```

Comment:
Instead of doing `trackCustomEvent` I think you can just do something like:

```
<OutboundLink href="https://www.gatsbyjs.com"
eventCategory="Gateways to Self-Serve" eventLabel="loc:banner; text:Gatsby Cloud!">
```

From the source code of the plugin: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-plugin-google-analytics/src/index.js#L17

Reply:
Woah! That's helpful! Looks like refactoring to use the Outbound Link that way might avoid some edge cases. Not sure that was documented in the plugin, should probably add that too..

Reply:
Depending on how you see this going, it might be useful to factor it into its own component like `<DotComLink>` or `<SelfServeGatewayLink>`?

It seems that there are some invariants:

1. the link is to gatsbyjs.com
2. You always want the same category (which is easy to mispell/typo)
3. Always want the text to be part of the label

Reply:
(Thank you Nat! <3)